### ISSUE TITLE
[0.8.2.0 staging-991] Tried to mute missing participant

### ISSUE THREAD

**Issue by SlayksWood** [StrangeLoopGames/EcoIssues#12508]
This case occurred once:

> Tried to mute missing participant 76561198072228212
UnityEngine.Logger:Log(LogType, Object)
UnityEngine.Debug:LogWarning(Object)
VoiceManager:SetPlayerMuted(String, Boolean)
VoiceChatItemUI:OnPointerDown(PointerEventData)
System.EventHandler`1:Invoke(Object, TEventArgs)
UnityEngine.EventSystems.ExecuteEvents:Execute(GameObject, BaseEventData, EventFunction`1)
UnityEngine.EventSystems.ExecuteEvents:ExecuteHierarchy(GameObject, BaseEventData, EventFunction`1)
UnityEngine.EventSystems.StandaloneInputModule:ProcessMousePress(MouseButtonEventData)
UnityEngine.EventSystems.StandaloneInputModule:ProcessMouseEvent(Int32)
UnityEngine.EventSystems.StandaloneInputModule:Process()
UnityEngine.EventSystems.EventSystem:Update()

I tried to mute another player, but I couldn't. We could talk, she could mute me.
See log file from 2167573 string.
Log file: https://drive.google.com/file/d/1STluxbGlbrv2DRk84xtyDISSWZ4oFSgw/view?usp=sharing

**Comment by redwyre** [comment]
@SlayksWood did you leave and join servers a few times before it happened?

**Comment by SlayksWood** [comment]
@redwyre 
Yes I reloaded game, then I few times disconnected and connected to server.
